README.md

```diff
@@ -5,11 +5,11 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/6f5885536c6b5c82f304/maintainability)](https://codeclimate.com/github/codus/codelog/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/6f5885536c6b5c82f304/test_coverage)](https://codeclimate.com/github/codus/codelog/test_coverage)
 
-A gem to help big teams to manage their chagelogs.
+A gem to help big teams manage their chagelogs.
```

Comment:
typo -> `chagelogs`

Reply:
Also, I might be  wrong but, i think the gem does a bit more than helping big teams managing their changelogs.

Reply:
```suggestion
A gem to help big teams manage their changelogs.
```